pyproject.toml

```diff
@@ -77,6 +77,7 @@ path = "src/awkward/_version.py"
 include = [
     "/src",
     "/tests",
+    "/tests/samples",
```

Comment:
@henryiii, the sdist test was failing because the pickle files that I made in Awkward v1 to test this weren't available. I could just include those files (`/tests/samples/*.pkl`), but all of the sample files should be included. I think it was passing tests before just because the sdist packaging test didn't have all optional dependencies installed. Therefore, it didn't test reading of Parquet and Avro files.

Adding the whole `/tests/samples` directory adds 220 kB. I think that's fine, even for an sdist. Do you have an opinion on that?

Reply:
I’d generally always include test files in SDist. Are these files gitignored or something?